mopidy_tidal/library.py

```diff
@@ -167,272 +49,79 @@ def get_distinct(self, field, query=None) -> set[str]:
                         apply_watermark(a.name)
                         for a in self._get_artist_albums(session, artist_id)
                     ]
-            elif field in {"track", "track_name"}:
+            elif field == "track":
```

Comment:
Duplicate condition check?

Reply:
This might be a consequence of trying to merge something I did not change but remained as part of much earlier fork.
Effectively is a "roll back" :)